sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPage.java

```diff
@@ -22,4 +22,23 @@
 
 public class EditContactPage {
   public static final By UUID = By.id("uuid");
+  public static final By USER_INFORMATION =
+      By.cssSelector(".v-slot.v-slot-h2.v-slot-vspace-top-none.v-slot-primary");
```

Comment:
I think we can change this performance

Reply:
discussed and we'll ask developers to help with a tag